src/FIN.ts

```diff
@@ -0,0 +1,14 @@
+import { shorten, genericCleanup, } from "./util";
+
+export class FIN {
+    private tag = "pb-fin";
+    cleanup() {
+        genericCleanup(this.tag);
+    }
+    run() {
+        const finlaHeader = document.querySelectorAll("table[class='t_JwOZxlVCPfN-FKRIoNo _2Fog1ad46aZ4q-RoEgK3R6 _1vWRpdI8cKNMPyOPnzlXgX _33A_5lETf4HBqwJi_q-jhZ'] > thead > tr > th");
```

Comment:
You probably don't need all of those class names in there. I'm guessing just one identifies the element, others are for styles.

Reply:
fixed